### ISSUE TITLE
Multiple language.

### ISSUE THREAD

**Issue by lovebdsobuj** [bluemix/NumberToArabicWordsAndroid#1]
Can you help me to add multiple language.

**Comment by bluemix** [comment]
yes, it is simple. copy the list of Arabic strings and replace the strings with the language you want.
They're are in the `ArabicTools.java`.

**Comment by lovebdsobuj** [comment]
I want to keep Arabic language and Want to add English language. 
User able to select English or Arabic language.

**Comment by bluemix** [comment]
yes, it can be translated to any language you want.
